activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -427,6 +427,8 @@ def new_column_definition(name, type, **options) # :nodoc:
 
       private
         def create_column_definition(name, type, options)
+          options.assert_valid_keys(:type, :force, :null, :default, :id, :precision, :scale, :array, :primary_key, :index, :polymorphic, :unique, :foreign_key, :limit, :first, :after)
```

Comment:
Failing tests with `:comment`, `:collation`, and `:auto_increment`

Reply:
👍 Good call